front/src/modules/ui/table/editable-cell/type/components/GenericEditableMoneyCell.tsx

```diff
@@ -0,0 +1,46 @@
+import { useRecoilValue } from 'recoil';
+
+import {
+  ViewFieldDefinition,
+  ViewFieldMoneyMetadata,
+} from '@/ui/editable-field/types/ViewField';
+import { EditableCell } from '@/ui/table/editable-cell/components/EditableCell';
+import { useCurrentRowEntityId } from '@/ui/table/hooks/useCurrentEntityId';
+import { tableEntityFieldFamilySelector } from '@/ui/table/states/selectors/tableEntityFieldFamilySelector';
+
+import { GenericEditableMoneyCellEditMode } from './GenericEditableMoneyCellEditMode';
+
+type OwnProps = {
+  viewField: ViewFieldDefinition<ViewFieldMoneyMetadata>;
+  editModeHorizontalAlign?: 'left' | 'right';
+};
+
+function formatNumber(x: number) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
What is this replace statement doing ? Can you leave a comment / add a named regex so the reader understands what this formatter does ?
related to https://github.com/twentyhq/twenty/issues/628 regarding how to format money amounts (not in the scope of this PR though)